### ISSUE TITLE
Do those normalization mean the same thing?

### ISSUE THREAD

**Issue by ashigirl96** [theogruner/Project_RL#3]
https://github.com/theogruner/rl_pro_telu/blob/e750cf7bc3fda4550b30006324a74b1b14d2150b/mpo/mpo.py#L294

<img width="854" alt="Screen Shot 2019-11-18 at 2 12 51" src="https://user-images.githubusercontent.com/29683674/69010994-fb178600-09a8-11ea-923e-93aa86734fd3.png">

Hello! I found your nice implementation about MPO.
well, I guess that should not implement as `torch.sum`, but use `torch.mean`.
Is my understanding correct?

**Comment by theogruner** [comment]
Hi, I really appreciate your interest in our implementation. You are totally right.
Since we do Monte Carlo estimation of the expectation, it should be the mean.
